src/app/utils/route-utils.ts

```diff
@@ -0,0 +1,12 @@
+import { ParaTime } from '../../config'
+
+export abstract class RouteUtils {
```

Comment:
To infer usage. `abstract` is here so that utils are not used as `new RouteUtils` - creating an instance of the class(which is not necessary), but rather inferring that it should be used as `RouteUtils.<static-method>`